translations/messages+intl-icu.de.yml

```diff
@@ -680,10 +680,24 @@ consultation.notification.mail.on.token.creation.confirmation: "Ein Zugangscode
 consultation.notification.mail.on.token.creation.error: "Es konnte keine Email versendet werden."
 consultation.notification.mail.subject.on.token.creation: "Ihre Stellungnahme zum Verfahren {procedureName} ist eingegangen"
 
+contact: "Kontakt"
 contact.data: "Kontaktdaten"
+contact.add: "Kontakt hinzufügen"
+contact.change: "Kontakt bearbeiten"
+contact.delete: "Kontakt löschen"
+contact.deleted: "Der Kontakt wurde gelöscht."
+contact.new: "Neuen Kontakt anlegen"
+contact.no: "Keine Kontakte angelegt"
```

Comment:
Could we tweak this translation key a little bit. Imho I think the key `contact.no` is not really transporting the translation message. Maybe something like `contact.no_entries` or `contact.no_new_entries`?

Reply:
good catch b9dbaffc0a16edd4bd35e58abdaac2bac8a3f44b